velox/substrait/SubstraitToVeloxPlan.h

```diff
@@ -107,6 +112,30 @@ class SubstraitVeloxPlanConverter {
   /// name>:<arg_type0>_<arg_type1>_..._<arg_typeN>
   const std::string& findFunction(uint64_t id) const;
 
+  /// Substrait emit feature integration. Here a given substrait::Rel is passed
+  /// and check if emit is defined for this relation. If so, the emit fields
+  /// are extracted from the relation. To simulate an emit, a projection is used
+  /// to pick the fields which is selected to be emitted.
+
+  core::PlanNodePtr ProcessEmit(
```

Comment:
naming: method names start with lower case; please, fix throughout

Please, move method implementation to the .cpp file to speed up compilation.

Reply:
I have renamed and moved the implementation to the .cpp and just kept the definition in .h since we are using a class method.